setup.py

```diff
@@ -68,7 +68,10 @@
         ],
         "dffml.port": ["json = dffml.port.json:JSON"],
         "dffml.service.cli": ["dev = dffml.service.dev:Develop"],
-        "dffml.config": ["json = dffml.config.json:JSONConfigLoader"],
+        "dffml.config": [
+            "json = dffml.config.json:JSONConfigLoader",
+            "png = config.png.dffml_config_png.config:PNGConfigLoader",
```

Comment:
We don't need to edit the top level `setup.py` since you've added it to the one in the plugin :)